### PR TITLE
Added Method Docs

### PR DESCRIPTION
**Pull request by avichal04**
Wrote documentation for some methods in scheduling.py

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (marcpage) [inline .idea/.gitignore:1]: Are you intending to add the the .idea/* files to this commit? What benefits does it add?

File: src/scheduling.py [hunk 2]
```diff
@@ -40,16 +40,17 @@ def logout():
         response.set_cookie(USER_ID_COOKIE, "", expires=0)
         return response
 
-    @app.route("/login", methods=["POST"])
+    @app.route("/login", methods=["POST"])  # using POST method for user login
```
Comment (marcpage) [inline src/scheduling.py:43]: Method documentation is typically in doc strings. Can you move these into triple quote comments directly inside the function?
def login():
   """using POST method for user login"""

File: src/scheduling.py [hunk 3]
```diff
@@ -59,7 +60,7 @@ def login():
     # Mark: User Actions
 
     @app.route("/create_user", methods=["POST"])
-    def create_user():
+    def create_user():  # Creates user
```
Comment (marcpage) [inline src/scheduling.py:63]: In the documentation were are starting to refer to users as employees. At some point we will update the code to refer to them as employees

**Review by marcpage — CHANGES_REQUESTED** [review]
Thank you so much for contributing. We are at a very early stage and are excited to have your support.

File: src/scheduling.py [hunk 4]
```diff
@@ -40,16 +40,17 @@ def logout():
         response.set_cookie(USER_ID_COOKIE, "", expires=0)
         return response
 
-    @app.route("/login", methods=["POST"])
+    @app.route("/login", methods=["POST"])  # using POST method for user login
     def login():
-        email = request.form["email"]
+        email = request.form["email"]  # Request module is used to fetch users "email" and "password"
```
Comment (marcpage) [inline src/scheduling.py:45]: We've also discovered that our PR lint checks are not failing. Please run. Black version 21.12b0 on the source before submitting and then run pylint version 2.12.2 and flake8 version 4.0.1 to verify it is clean. 

On a macOS system, you should be able to run `./pr_build.sh fix` to do all these actions.

file would be reformatted, 1 file would be left unchanged.

************* Module scheduling
src/scheduling.py:45:0: C0301: Line too long (101/100) (line-too-long)
src/scheduling.py:48:0: C0301: Line too long (113/100) (line-too-long)
src/scheduling.py:83:0: C0301: Line too long (115/100) (line-too-long)
src/scheduling.py:101:0: C0303: Trailing whitespace (trailing-whitespace)
src/scheduling.py:93:8: W0612: Unused variable 'user' (unused-variable)
src/scheduling.py:172:4: E0102: function already defined line 92 (function-redefined)

**Review by marcpage — CHANGES_REQUESTED** [review]

File: src/scheduling.py (hunk 2 above)
Comment (avichal04) [reply src/scheduling.py:43]: Yes I will change it to triple quote comments.
Actually this is my first time contributing to open source.
So I hope you understand it.

File: src/scheduling.py (hunk 3 above)
Comment (avichal04) [reply src/scheduling.py:66]: Ok I was unaware of that, will now change it to employees.

File: src/scheduling.py (hunk 2 above)
Comment (marcpage) [reply src/scheduling.py:43]: No worries, we're happy to have you contribute.

File: src/scheduling.py (hunk 3 above)
Comment (marcpage) [reply src/scheduling.py:66]: Don't worry, there was no way for you to know.

File: src/scheduling.py (hunk 2 above)
Comment (avichal04) [reply src/scheduling.py:43]: Thank you.

Comment (avichal04) [reply src/scheduling.py:43]: I wanted to know that should I comment in triple quotes only?

File: .idea/.gitignore (hunk 1 above)
Comment (avichal04) [reply .idea/.gitignore:1]: No.

File: src/scheduling.py (hunk 4 above)
Comment (avichal04) [reply src/scheduling.py:45]: git checkout my_name/meaningful_change_name
can you help me in this.
I am little bit confused how to do it.

Comment (avichal04) [reply src/scheduling.py:45]: What is to be written in "my_name"

Comment (avichal04) [reply src/scheduling.py:45]: what is to be written in "meaningful_change_name"

**Comment by marcpage** [comment]
This work was done in another pull request. Closing out this duplicate one.

**Pull request closed without merging**